ssl-checker/network_utils.py

```
@@ -76,16 +77,126 @@ def resolve_domain_to_ip(domain: str) -> str:
+        url = IP_API_COM_URL.format(ip=ip)
+        response = requests.get(url, timeout=CONNECTION_TIMEOUT)
+        response.raise_for_status()
+        data = response.json()
+        
+        # Check if the API returned success status
+        if data.get('status') == 'success':
+            return _normalize_ip_api_com_response(data)
+        # If status is not success (e.g., 'fail'), fall through to ipinfo.io
+    except Exception:
+        # Fall through to ipinfo.io on any error
+        pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)